etl/steps/data/garden/un/2024-04-09/undp_hdr.meta.yml

```diff
@@ -0,0 +1,463 @@
+definitions:
```

Comment:
For the previous version (`2022-11-29`) I added additional metadata in the new format in the grapher step. It would be good to integrate that to this file, since it was agreed in the HDI redesign.

The differences are mostly around the topic tags and the metadata and grapher config for HDI.